public/app/features/canvas/runtime/sceneAbleManagement.ts

```diff
@@ -269,13 +269,11 @@ export const initMoveable = (destroySelecto = false, allowChanges = true, scene:
           }
         }
 
-        if (!config.featureToggles.canvasPanelPanZoom) {
-          targetedElement.tempConstraint = { ...targetedElement.options.constraint };
-          targetedElement.options.constraint = {
-            vertical: VerticalConstraint.Top,
-            horizontal: HorizontalConstraint.Left,
-          };
-        }
+        targetedElement.tempConstraint = { ...targetedElement.options.constraint };
```

Comment:
I'm not sure of the original purpose of this conditional `if (!config.featureToggles.canvasPanelPanZoom)` which appears in multiple places.